<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowWeaponEncoder</class>
 <widget class="QMainWindow" name="MainWindowWeaponEncoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowWeaponEncoder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1471</width>
      <height>881</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout" stretch="2,0,3">
     <item>
      <layout class="QVBoxLayout" name="weaponEntryVert">
       <item>
        <layout class="QHBoxLayout" name="LineEditLayout">
         <item>
          <layout class="QVBoxLayout" name="IdLayout">
           <item>
            <widget class="QLabel" name="IdLabel">
             <property name="text">
              <string>Id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="IdEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="nameLayout">
           <item>
            <widget class="QLabel" name="NameLabel">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="NameEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="valueLayout" stretch="0,1,1,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="dropdownButton" name="lockButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="attackLayout" stretch="1,2,0">
           <item>
            <widget class="QLabel" name="atkLabel">
             <property name="text">
              <string>Attacks</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="AttackEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="VarAtkLayout">
             <item>
              <spacer name="VarStkSpcL">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="randomAttacks">
               <property name="text">
                <string>Random</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="randomAttackDiceCount"/>
             </item>
             <item>
              <widget class="dropdownButton" name="randomAttackDiceType">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="VarAtkSpcR">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="DamageLayout" stretch="1,2,0">
           <item>
            <widget class="QLabel" name="dmgLabel">
             <property name="text">
              <string>Damage</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="DmgEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="VarDmgLayout">
             <item>
              <spacer name="VarDmgSpcL">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="VarDmgCheck">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Variable</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="VarDmgSpcR">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="dropdownButton" name="facingButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Special Rules:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="dropdownButton" name="specialSelect">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="specialX"/>
         </item>
         <item>
          <widget class="QPushButton" name="specialAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="specialReset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="specialView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="listLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="SaveWeaponButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="rosterLabel">
         <property name="text">
          <string>Roster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="weaponView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dropdownButton</class>
   <extends>QToolButton</extends>
   <header location="global">dropdownbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
